app/controllers/credentials_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
   def callback
     webauthn_credential = relying_party.verify_registration(
       params,
-      session["current_registration"]["challenge"],
+      session[:current_registration][:challenge],
```

Comment:
@soartec-lab it turns out that it was needed to change access the session data using `symbol` keys instead of `strings` as it was in fact returning `nil` after upgrading to Rails 7.0 😅

Reply:
@santiagorodriguez96 

Thank you for let me know. My doubts were cleared. And I'm glad I was able to contribute👍